assignment1/kaprekar.rb

```diff
@@ -0,0 +1,15 @@
+def kaprekar?(k)
+    ks = k**2
+    lenf = (ks.to_s.length)-1
+    lenh = (ks.to_s.length / 2) - 1
+    a = ks.to_s[0..lenh].to_i
+    b = ks.to_s[lenh+1..lenf].to_i
+    k == (a+b)
+    puts ks
+    puts lenf
+    puts lenh
+    puts a
+    puts b
+end
+
```

Comment:
Write this function in the way that it should return "Kaprekar Number" if it is Karprekar number and "Not Karprekar number" otherwise